src/style.css

```diff
@@ -84,7 +110,3 @@ div#mathlive-suggestion-popover.bottom-tip::after {
   border-top-color: var(--ls-primary-background-color);
 }
 
-li.ML__popover__current {
-  background-color: var(--ls-selection-background-color);
-  color: var(--ls-selection-text-color);
-}
```

Comment:
These were intended to make the highlighted selection match the current theme. Is there any reason to delete them?